libcontainer/integration/template_test.go

```diff
@@ -22,8 +22,9 @@ var standardEnvironment = []string{
 const defaultMountFlags = unix.MS_NOEXEC | unix.MS_NOSUID | unix.MS_NODEV
 
 type tParam struct {
-	userns  bool
-	systemd bool
+	userns      bool
+	systemd     bool
+	personality int
```

Comment:
Unless it's used in a few tests, I wouldn't add this as a parameter.

As it is used in 1 test only, better to set it in the test itself.

Reply:
@kolyshkin Thanks this is resolved in latest commit